0_intro/exercises/bubble_sort/src/lib.rs

```diff
@@ -1,5 +1,13 @@
 pub fn bubble_sort(v: &mut Vec<i32>) {
-    todo!()
+    while !is_sorted(v){
+        for i in 0..(v.len() - 1){
```

Comment:
There is a missing ' ' between the ')' and '{'. You should use [rustfmt](https://github.com/rust-lang/rustfmt#quick-start)